src/components/input/input.svelte

```diff
@@ -133,6 +134,13 @@
     value = e.currentTarget['value']
     hasErrorsInternal = (required && !value) || !input?.checkValidity()
   }
+
+  let tabindex : number;
+  
+  onMount(() => {
+    tabindex = input.getRootNode() === document ? $$restProps.tabindex : 1;
```

Comment:
oh nice, I hadn't considered passing through `tabindex` from `$$restProps`! One suggestion, lets check its a `leo-input` in particular, in case we render this inside a different shadowRoot.

```suggestion
    tabindex = input.getRootNode().host?.tagName !== 'LEO-INPUT' ? $$restProps.tabindex : 1;
```

Reply:
Oh nice, yeah thanks for pointing that out. Done!

Went about it this way, think it reads a little better:

```ts
  onMount(() => {
    const inputRootNode = input.getRootNode() as ShadowRoot;

    tabindex = inputRootNode.host?.tagName === 'LEO-INPUT' ?  1 : $$restProps.tabindex;
  });

```